instructors/instructor-notes.md

```diff
@@ -153,30 +159,15 @@ you will be asked for a 6-digit host key, which you can find [in this message](h
 See [the explanation here][zoom-host-key] for step-by-step instructions on how to
 claim host in Zoom Rooms using the host key.
```

Comment:
@ndporter if you'd like to add a link to the [zoom documentation in the handbook](https://docs.carpentries.org/topic_folders/communications/tools/zoom_rooms.html#zoom-rooms), you can do that. However, I think we (CT) decided our goal was to cut back on the instructional content on our end because this adds to the maintenance load for our documentation when Zoom makes changes. So I think the most sustainable choice is to link only to instructions that are hosted and maintained by Zoom.